src/gutenberg2zim/export.py

```diff
@@ -821,7 +824,7 @@ def handle_companion_file(
                     as_ext=".zip",
                 )
             else:
-                Path(tmp_epub.name).move(str(dst))
+                tmp_epub.rename(Path(dst) / tmp_epub.name)
                 Global.add_item_for(path=dstfname, fpath=dst)
```

Comment:
`dst` is already a Path

```suggestion
                tmp_epub.rename(dst / tmp_epub.name)
```